modules/ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod.adoc

```diff
@@ -0,0 +1,81 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod_{context}"]
+= Using a SharedConfigMap instance in a pod
+
+[role="_abstract"]
+To access a `SharedConfigMap` custom resource (CR) instance from a pod, you grant a given service account RBAC permissions to use that `SharedConfigMap` CR instance.
+
+.Prerequisites
+
+* You have created a `SharedConfigMap` CR instance for the config map that you want to share across namespaces in the cluster.
+* You must have permission to perform the following actions:
+** Discover which `SharedConfigMap` CR instances are available by entering the `oc get sharedconfigmaps` command and getting a non-empty list back.
+** Determine if the service account your pod specifies is allowed to use the given `SharedSecret` CR instance. That is, you can run the `oc adm policy who-can use <identifier_of_specific_sharedsecret>` command to see if the service account in your namespace is listed.
+** Determine if the service account your pod specifies is allowed to use `csi` volumes, or if you, as the requesting user who created the pod directly, are allowed to use `csi` volumes.  See "Understanding and managing pod security admission" for details.
+
+[NOTE]
+====
+If neither of the last two prerequisites in this list are met, create, or ask someone to create, the necessary role-based access control (RBAC) so that you can discover `SharedConfigMap` CR instances and enable service accounts to use `SharedConfigMap` CR instances.
+====
+
+.Procedure
+
+. Grant a given service account RBAC permissions to use the `SharedConfigMap` CR instance in its pod by using the `oc apply` command with YAML content.
++
+[NOTE]
+====
+Currently, the `kubectl` command and the OpenShift CLI (`oc`) have hard-coded, special case logic restricting the `use` verb to roles centered around pod security. Therefore, you cannot use `oc create role` commands to create the role needed for consuming a `SharedConfigMap` CR instance.
+====
++
+[source,terminal]
+----
+oc apply -f - <<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: shared-resource-my-share
+  namespace: my-namespace
+rules:
+  - apiGroups:
+      - sharedresource.openshift.io
+    resources:
+      - sharedconfigmaps
+    resourceNames:
+      - my-share
+    verbs:
+      - use
+EOF
+----
```

Comment:
I recommend moving this up as "step 2" in the procedure for creating the SharedSecret and SharedConfigMap